### ISSUE TITLE
[Feb 2025] Accessibility Audit: Static site Misc sections

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8509]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the pages in the following miscellaneous sections of the static site that have been updated in the last 3 months. Do the page scan as well as any relevant guided tests.
Pages to visit:

- [x] https://www.simplereport.gov/
- [x] https://www.simplereport.gov/about-us/
- [x] https://www.simplereport.gov/covid-19-results/
- [x] https://www.simplereport.gov/contact-us
- [x] https://www.simplereport.gov/support/
- [x] https://www.simplereport.gov/exit-disclaimer/
- [x] https://www.simplereport.gov/terms-of-service/
- [x] https://www.simplereport.gov/unsupported-browser/
- [x] https://www.simplereport.gov/waitlist/ 

Components to test:
- [x] Header
- [x] Footer
- [x] Touchpoints modal

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audit #6337 #5768

**Comment by emyl3** [comment]
Updated the description to include checkboxes to keep track of completed tasks easier :D

**Comment by emyl3** [comment]
Issues created:
- https://app.zenhub.com/workspaces/simplereport-2025-and-onwards-605b43a6ceb92c000f86279a/issues/gh/cdcgov/prime-simplereport-site/788
- https://app.zenhub.com/workspaces/simplereport-2025-and-onwards-605b43a6ceb92c000f86279a/issues/gh/cdcgov/prime-simplereport-site/789
- https://app.zenhub.com/workspaces/simplereport-2025-and-onwards-605b43a6ceb92c000f86279a/issues/gh/cdcgov/prime-simplereport-site/790

**Comment by emyl3** [comment]
Error findings:
- dev.simplereport.gov/ + touchpoints modal - [202502-dev.simplereport.gov-touchpoints_modal.csv](https://github.com/user-attachments/files/18950216/202502-dev.simplereport.gov-touchpoints_modal.csv)
